src/Mod/PartDesign/App/json_fwd.hpp

```diff
@@ -0,0 +1,78 @@
+#ifndef INCLUDE_NLOHMANN_JSON_FWD_HPP_
+#define INCLUDE_NLOHMANN_JSON_FWD_HPP_
+
+#include <cstdint> // int64_t, uint64_t
+#include <map> // map
+#include <memory> // allocator
+#include <string> // string
+#include <vector> // vector
+
+/*!
+@brief namespace for Niels Lohmann
+@see https://github.com/nlohmann
+@since version 1.0.0
```

Comment:
In the forum you said this file is under MIT but this header file lacks of this information. So, it's best to directly add the license here, too.

According to the doxygen documentation a tag \license doesn't exist but apparently \copyright is used for this purpose.
So, `\copyright MIT` should be OK.

Reply:
I contacted upstream with this:

> The MIT license not really requires adding the license to all files, but I may work on this in the future.

https://github.com/nlohmann/json/discussions/2454